Added '-p' / '--parse' to automatically parse the input file

0xeb opened the pull request.

0xeb committed:
Added '-p' / '--parse' to automatically parse the input file

FraMuCoder commented:
Thanks for your pull request. I would like to merge it but I want to merge it in develop branch first. Than I can also update documentation before increasing version.
I see no option to change target branch on my own so maybe you can do it? Otherwise I would merge it on command line and close this pull request.

FraMuCoder commented:
I pushed it to develop and if I merge develop do master this pull request should be closed.

0xeb committed:
+ added parse options

This allows passing options such as:

- Default parsing
- Detailed processing record
- Incomplete parsing
- etc.
